kategory/src/main/kotlin/kategory/instances/ListKWInstances.kt

```diff
@@ -0,0 +1,76 @@
+package kategory
+
+interface ListKWInstances :
+        Functor<ListKW.F>,
+        Applicative<ListKW.F>,
+        Monad<ListKW.F>,
+        Traverse<ListKW.F> {
+
+    override fun <A> pure(a: A): ListKW<A> =
+            ListKW.listOfK(a)
```

Comment:
Detekt is asking for these to be in the same line. This is something I dislike but until we fix the detekt rules this should be :
```kotlin
override fun <A> pure(a: A): ListKW<A> = ListKW.listOfK(a)
```